src/tests/FunctionalTests/Android/Device_Emulator/JIT/Program.cs

```diff
@@ -8,6 +8,7 @@ public class Program
     public static int Main(string[] args)
     {
         Console.WriteLine("Hello, Android!"); // logcat
-        return 42;
+        int result = ((string)AppContext.GetData("System.Threading.ThreadPool.MinThreads")).Equals("4") ? 42 : 1;
```

Comment:
I wonder if it's better to set and test some more arbitrary property, since `System.Threading.ThreadPool.MinThreads` will probably have a an affect on something real in the threadpool

Reply:
Created some random string.